lsp-bridge.el

```diff
@@ -1490,11 +1492,16 @@ So we build this macro to restore postion after code format."
                (lsp-bridge-process-live-p))
       (unless (or (string-equal current-word "") (null current-word))
         (if (lsp-bridge-is-remote-file)
-            (lsp-bridge-remote-send-func-request "ctags_complete"
-                                                 (list
-                                                  current-word
-                                                  (file-local-name (buffer-file-name))
-                                                  (1- (point))))
+            ;; remote file buffer do not associate with an actual file on the disk
+            ;; the buffer is created by lsp-bridge-opne-remote-file--response
+            ;; hence (buffer-file-name) will return nil
+            ;; should use buffer local variable lsp-bridge-remote-file-path
+            (with-current-buffer (buffer-name)
+              (lsp-bridge-remote-send-func-request "ctags_complete"
```

Comment:
Why need wrap ```with-current-buffer (buffer-name)```?

Reply:
As I explaied in the comment, in my environment (emacs 29.1 macOS), `(buffer-file-name)` sometimes will return nil, which cause `(file-local-name (buffer-file-name))` to raise an error and interrupt the whole process.

when error happened `*Message*` buffer outputs `file-local-name: Wrong type argument: stringp, nil`

The underlying problem is that remote file buffer is a virtual buffer created by `lsp-bridge-opne-remote-file--response` using 
```elisp
(let ((buf-name (format "[LBR] %s" (file-name-nondirectory path))))
    (lsp-bridge-define--jump-record-postion)

    (with-current-buffer (get-buffer-create buf-name)

```

as you can see it use `get-buffer-create`, it is not directly related to a file on disk, so (buffer-file-name) returns nil.

`lsp-bridge-remote-file-path` is a buffer local variable which related to the current buffer, by using `(with-current-buffer (buffer-name)` we can ensure that we are accessing the buffer local variable `lsp-bridge-remtoe-file-path` of current buffer.

After this change, the issue is resolved, i can get the completion menu popup successfully all the time. so I think it is the where the issue occurs.